felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/OpprettTilbakekrevingRequest.kt

```diff
@@ -30,7 +30,9 @@ data class OpprettTilbakekrevingRequest(
     @field:Valid
     val faktainfo: Faktainfo,
     @field:Valid
-    val institusjon: Institusjon? = null
+    val institusjon: Institusjon? = null,
+    @field:Valid
+    val manuelleBrevmottakere: Set<Brevmottaker> = emptySet()
```

Comment:
Kan være vi ikke trenger å ha denne som en liste, hvis brevmottakere av type fullmektig og verge i ba-sak begge skal mappes til eksisterende Verge-klasse i familie-tilbake
```suggestion
    val manuellBrevmottaker: Brevmottaker? = null
```

Reply:
tror vi må ha det som en liste da det kunne hende man registrerte flere manuelle bravmottakere i ba-sak.